### PR TITLE
created profile model

### PR DESCRIPTION
**Pull request by DerinAde**
I created the profile model, saw Matt's suggestion on slack, Dan please review and I'll make changes.

### CODE REVIEW

File: server/profile.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from app import db
+
+class Profile(db.Model):
+    __tablename__ = 'profile'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    image = db.Column(db.Text)
```
Comment (sina-jamshidi) [inline server/profile.py]: we can have a more descriptive name like `profile_image` or `profile_picture_url`, etc.

File: server/profile.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    price = db.Column(db.Float(precision=2), nullable=False)
+    ingredients = db.Column(db.String(200), nullable=True)
+    portion = db.Column(db.Integer, nullable=False)
+    required_tools = db.Column(db.String(200), nullable=True)
```
Comment (sina-jamshidi) [inline server/profile.py]: I think these are recipe fields. Revisit the mockups and think about the database design in terms of the relationship between users/chefs/recipes and what information is needed in which model.

**Review by sina-jamshidi — CHANGES_REQUESTED** [review]

File: server/profile.py (hunk 1 above)
Comment (DerinAde) [reply server/profile.py]: i'll update the name fields

**Review by dproc96 — APPROVED** [review]
This looks good, nice work!

**Pull request merged**